uniffi_bindgen/src/bindings/kotlin/templates/CustomType.kt

```diff
@@ -0,0 +1,43 @@
+object FfiConverterType{{ name }} {
+    {%- match config %}
+    {%- when None %}
+    {#- No custom type config, just forward all methods to our builtin type #}
```

Comment:
Logic in the template which affects almost the whole of the file indicate we should probably have split this template in two.

Reply:
Agreed, although it's not so easy to do this currently.  I'll try to refactor this as part of #1089.